com.unity.render-pipelines.universal/Shaders/Utils/StencilDeferred.shader

```diff
@@ -169,6 +170,11 @@ Shader "Hidden/Universal Render Pipeline/StencilDeferred"
                         unityLight.shadowAttenuation = MainLightShadow(shadowCoord, posWS.xyz, shadowMask, _MainLightOcclusionProbes);
                     #endif
                 }
+
+                #if defined(_MAIN_LIGHT_COOKIE)
+                    half3 cookieColor = URP_LightCookie_SampleMainLightCookie(posWS);
```

Comment:
This line will introduce a shader warning here when real is defined as float since the function returns a real3.

Reply:
Thanks. I will fix it.